rasa/core/policies/default_flows.yml

```diff
@@ -72,10 +72,6 @@ flows:
       - id: "1"
         action: utter_corrected_previous_input
       - id: "2"
-        set_slots:
```

Comment:
as discussed in sprint planning, let's replace this with a simpler `pattern_correction` flow that doesn't do the `confirm_correction` step. And then copy this current version with the confirmation into the docs, to show users how they can override the default flows if they want to add a confirmation step.